pkgs/build-support/dotnet/add-nuget-deps/default.nix

```diff
@@ -68,6 +101,8 @@ attrs
             isExecutable = true;
             inherit cacert;
             nugetToNix = nuget-to-nix;
+            nugetToJson = nuget-to-json;
+            jq = jq;
             nixfmt = nixfmt-rfc-style;
           };
```

Comment:
General cleanup idea: Could we pass the exes in via a bin-path instead of explicitly substituting them? Like how its done in `nuget-to-json`.

Reply:
Done.